jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/rabbitmq/RabbitMqDocumentInput.java

```diff
@@ -61,12 +61,12 @@ public void receiveMessage(DocumentReadyMessage documentReadyMessage) throws Doc
 		
 		this.documentReadyHandler.handle(content, documentReadyMessage.getTransactionInfo());
 
-                logger.debug("attempting to delete the document from redis storage");
+        logger.debug("attempting to delete the document from redis storage");
 
-                //when successfully processed, delete the document from redis storage
-                this.redisStorageService.deleteString(storageProperties.getKey());
+        //when successfully processed, delete the document from redis storage
+        this.redisStorageService.deleteString(storageProperties.getKey());
 
-                logger.info("document successfully deleted from redis storage");
+        logger.info("document successfully deleted from redis storage");
```

Comment:
looks like maybe some spacing got affected in this file... could be worth fixing :)